src/v/cluster/cluster_utils.h

```diff
@@ -40,6 +40,23 @@ patch<broker_ptr> calculate_changed_brokers(
 
 /// Creates the same topic_result for all requests
 // clang-format off
+template<typename T>
```

Comment:
Please add more details to commit tittle and PR title f.e. `c/topics_frontend: dispatching create topics to leader`